### DISCUSSION TITLE
[Project Proposal Review]Async Api Diff

### DISCUSSION THREAD

**Discussion by Souvikns** [postmanlabs/gsoc#37] [Ideas]
Hi, I am Souvik De, a 3rd-year undergrad pursuing my BE-CSE degree from Chandigarh University. 

I am interested in working on idea 9 
Here is my proposal:
https://docs.google.com/document/d/1yOrZsCRxtJhB6p--X7ucKkdd7zPrUsb9wPPW3WP1sRs/edit?usp=sharing
> Please have a look thanks, I also have some doubts about the timeline, any suggestion would be very helpful.

**Comment by Sharath-Postman** [comment]
Thank you so much for writing in. Please do let me know if you haven't received any review yet.

Cheers

**Reply by Souvikns** [reply]
Yeah, my proposal was reviewed, I am working on it and updating it accordingly.

**Reply by vinitshahdeo** [reply]
@Souvikns A gentle reminder to make the submission before the deadline which ends today. Ignore if you have submitted already. All the best. :v:

**Reply by Souvikns** [reply]
@vinitshahdeo thanks for the reminder, I submitted it yesterday.
